flink-runtime/src/test/java/org/apache/flink/runtime/testutils/ZooKeeperTestUtils.java

```diff
@@ -37,6 +41,33 @@ public class ZooKeeperTestUtils {
 
     private static final Logger LOG = LoggerFactory.getLogger(ZooKeeperTestUtils.class);
 
+    /**
+     * Creates a new {@link TestingServer}, setting additional configuration properties for
+     * stability purposes.
+     */
+    public static TestingServer createZookeeperTestingServer() throws Exception {
+        return new TestingServer(getZookeeperInstanceSpecWithIncreasedSessionTimeout(), true);
+    }
+
+    private static InstanceSpec getZookeeperInstanceSpecWithIncreasedSessionTimeout() {
+        final InstanceSpec instanceSpec = InstanceSpec.newInstanceSpec();
+
+        final Map<String, Object> properties = new HashMap<>();
+        properties.put("maxSessionTimeout", 60_000);
+
+        return new InstanceSpec(
+                instanceSpec.getDataDirectory(),
+                instanceSpec.getPort(),
+                instanceSpec.getElectionPort(),
+                instanceSpec.getQuorumPort(),
+                instanceSpec.deleteDataDirectoryOnClose(),
+                instanceSpec.getServerId(),
+                instanceSpec.getTickTime(),
+                instanceSpec.getMaxClientCnxns(),
+                properties,
+                instanceSpec.getHostname());
```

Comment:
```suggestion
        final boolean deleteDataDirectoryOnClose = true;
        return new InstanceSpec(
                instanceSpec.getDataDirectory(),
                instanceSpec.getPort(),
                instanceSpec.getElectionPort(),
                instanceSpec.getQuorumPort(),
                deleteDataDirectoryOnClose,
                instanceSpec.getServerId(),
                instanceSpec.getTickTime(),
                instanceSpec.getMaxClientCnxns(),
                properties,
                instanceSpec.getHostname());
```
I would make this parameter more explicit (it's actually true in the default `InstanceSpec` we use) to improve the readability of the code since this is a more important property (in contrast to the other ones) here.